README.md

```diff
@@ -62,6 +62,7 @@ List of some daily used commands for tech geeks.
 - [Mongo](commands/mongo#mongo-point_left)
     - [Dump collection from mongoDB](commands/mongo/mongo-dump-collection.md)
     - [Restore collection to mongoDB](commands/mongo/mongo-restore-collection.md)
+    - [Dump database from mongoDB](commands/mongo/mongo-dump-database.md)
```

Comment:
Can you add this,  in this file also
https://github.com/arshadkazmi42/ak-cli/tree/master/commands/mongo#mongo-point_left